Cargo.toml

```diff
@@ -17,16 +17,18 @@ include = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["std"]
+default = ["std", "use-blake2b-rs"]
```

Comment:
this is a breaking change, suggest to remove this feature flag from `default` set and add a new feature flag like `use-blake2b-ref`.

Reply:
could you try this?

```toml
[dependencies]
blake2b-rs = { optional = true }
blake2b-ref = { optional = true }

[features]
default = ["std", "with-blake2b-rs"]
with-blake2b-ref = []

with-blake2b-rs = ["dep:blake2b-rs"]
with-blake2b-ref = ["dep:blake2b-ref"]
```

Reply:
A default blake2b is needed here, otherwise the user will report an error when using `--no-default-features`.

Reply:
> --no-default-features

I think it can be resolved by using `--no-default-features --features with-blake2b-rs` which is better than breaking the default features set compatibility.

Reply:
OK